tests/integration/install/test_installation.py

```diff
@@ -131,17 +131,15 @@ def test_job_cluster_policy(ws, installation_ctx):
 @retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=5))
 def test_running_real_remove_backup_groups_job(ws: WorkspaceClient, installation_ctx: MockInstallationContext) -> None:
```

Comment:
Test that looked to be unrelated to the version changes failed in [CI](https://github.com/databrickslabs/ucx/actions/runs/11930043705). Note we have seen this test to be flaky due to the eventual consistency of the group API. These changes force the test to be ran as part of the integration test in the PR